library/src/main/java/com/devbrackets/android/exomedia/core/video/ResizingTextureView.java

```diff
@@ -331,9 +338,7 @@ public void setVideoRotation(@IntRange(from = 0, to = 359) int userRotation, @In
         requestedUserRotation = userRotation;
         requestedConfigurationRotation = configurationRotation;
 
-        if (matrixManager.ready()) {
-            matrixManager.rotate(this, (userRotation + configurationRotation) % MAX_DEGREES);
-        }
+        updateRotateScale();
```

Comment:
Hmm, We have two options then:
1. Pass in the `TextureView` as an additional param to `setIntrinsicVideoSize`
2. Store the `TextureView` as a `WeakReference` when the `matrixManager` isn't ready (and clean up after we perform the requested updates in `setIntrinsicVideoSize`)

Both have positives and negatives, but I think I'm leaning towards the second option